packages/js/src/MatomoTracker.ts

```diff
@@ -21,6 +21,8 @@ class MatomoTracker {
 
   // Initializes the Matomo Tracker
   static initialize({ urlBase, siteId, trackerUrl, srcUrl }: UserOptions) {
+    if (!urlBase.endsWith('/')) urlBase = urlBase + '/'
```

Comment:
Could you use a string with index here instead? Unfortunately, we still have to support users that are on IE11 (Windows 7) and since this API is not supported there and the build system does not add a polyfill I am afraid this will break.

```suggestion
    if (urlBase[urlBase.length - 1] !== '/') urlBase = urlBase + '/'
```